src/main/java/com/example/petbuddybackend/utils/exception/advice/GeneralAdvice.java

```diff
@@ -12,15 +18,18 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
+@Slf4j
 @RestControllerAdvice
 public class GeneralAdvice {
 
     @ExceptionHandler(Throwable.class)
     @ResponseStatus(code = HttpStatus.INTERNAL_SERVER_ERROR)
     public ApiExceptionResponse handleException(Throwable t) {
+        log.error("Unhandled exception", t);
         return new ApiExceptionResponse(t);
     }
 
+    @Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
potrzebne to?

Reply:
Tak, bez tego wyjątki mogą łapać się w tym advice z throweable, który powinien łapać wszyskie wyjątki które nie są w inny sposób handlowane.

Ale fakt, trochę się namnożyło tych Ordered więc rozdzieliłem te handlery na dwa pliki